tests/unit/plugins/filter/test_json_patch.py

```diff
@@ -0,0 +1,318 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) Stanislav Meduna (@numo68)
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type  # pylint: disable=C0103
+
+import unittest
+import json
+from ansible_collections.community.general.plugins.filter.json_patch import FilterModule
+from ansible.errors import AnsibleFilterError
+
+
+class TestJsonPatch(unittest.TestCase):
+    def setUp(self):
+        self.filter = FilterModule()
+        self.json_patch = self.filter.filters()["json_patch"]
+        self.json_diff = self.filter.filters()["json_diff"]
+        self.json_patch_recipe = self.filter.filters()["json_patch_recipe"]
+
+    # json_patch
+
+    def test_patch_add_to_empty(self):
+        result = json.dumps(
+            self.json_patch({}, "add", "/a", 1),
+            sort_keys=True,
+            separators=(",", ":"),
+        )
+        self.assertEqual(result, '{"a":1}')
```

Comment:
Why dump the result and comparing JSON strings, instead of comparing objects:
```suggestion
        self.assertEqual(result, {"a": 1})
```
?